cmd/hyperfed/main.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// A binary that can morph into all of the other kubefed binaries. You can
+// A binary that can morph into all of the other federation-v2 binaries. You can
```

Comment:
revert

Reply:
Fixed. That's strange as this wasn't a conflict.